src/condor_daemon_client/dc_transfer_queue.h

```diff
@@ -103,6 +103,8 @@ class DCTransferQueue : public Daemon {
 
 	bool GoAheadAlways( bool downloading ) const;
 
+	inline bool HasTransferPermission() const { return m_xfer_queue_sock && m_xfer_queue_go_ahead; }
```

Comment:
Is this deliberately unused in this PR?

Reply:
In converstation, ColeB suggests that this function will be useful in phase 1B and/or 2.